scripts/rewrite-decl-statements-for-closure-test.js

```diff
@@ -138,7 +138,13 @@ function transform(srcFile, rootDir) {
       }
 
       const variableDeclarator = t.variableDeclarator(variableDeclaratorId, callExpression);
-      path.replaceWith(t.variableDeclaration('const', [variableDeclarator]));
+      const variableDeclaration = t.variableDeclaration('const', [variableDeclarator]);
```

Comment:
Question, since I don't necessarily fully understand all the stuff we do in here:

Is this assuming that we will always have at least one import in a file, and relying on the first import to find the license comment? If so, will we encounter problems for files with no imports, e.g. */constants.js?

Reply:
If there are no imports to replace, then the entire 'ImportDeclaration' callback doesn't trigger...which means no path/node is replaces with a 'goog.require' version.

The replacing of a node was what caused the comment information to drop, because I didn't copy the comments onto the new node. So actually constants.js was fine, and had the license information in the transpiled version. But any file with import statements was losing the license information.

Reply:
Oh, I see. Thanks for explaining. Maybe we should add a comment to this code e.g. "Preserve comments when replacing imports"?